spec/functional/resource/user/windows_spec.rb

```diff
@@ -44,29 +46,138 @@ def delete_user(u)
     shell_out("net user #{u} /delete")
   end
 
-  before do
+  def backup_secedit_policy
+    backup_command = "secedit /export /cfg #{ENV['TEMP']}\\secedit_restore.inf /areas SECURITYPOLICY"
+    system(backup_command)
```

Comment:
are you intentionally using `system` instead of `shell_out`? personal preference or is there a reason that should be put in a comment?

Reply:
No preference, I'm just used to calling `system()` - I've switched them to `shell_out` for consistency.